db.go

```diff
@@ -12,6 +13,16 @@ var ErrNoDB = errors.New("no DB set")
 type DB interface {
 	// Save stores the provided TestResult in the data store and returns its unique ID.
 	Save(context.Context, TestResult) (string, error)
+	// Enumerate lists the test results for the given page. The datastore determines the page size.
+	Enumerate(ctx context.Context, page int) (results []Summary, more bool, err error)
+}
+
+type Summary struct {
+	ID      string
+	Started time.Time
+	Dur     time.Duration
+	Total   int
+	Passed  int
```

Comment:
gonna make us do math to know how many failed?!?!

Reply:
I would prefer failed instead of pass if had to choose.

Reply:
Why not both?